### PR TITLE
User option to disable/enable QnA notifications

### PR DESCRIPTION
**Pull request by dannimajdalani**
Closes [#3](https://github.com/vanilla/support/issues/3)

**What Changed**

I added notification preferences for QnA when:

- A question is answered 
- An answer is accepted

### CODE REVIEW

**Comment by cchabilall83** [comment]
I tested every scenario, it seemed to work.  I was getting some weird results, but I think I wasn't saving my preferences properly. @topcheese13 if you receive any inconsistent results let me know.

**Comment by cchabilall83** [comment]
also do the strings need to be translated?

**Comment by dannimajdalani** [comment]
> also do the strings need to be translated?

Yes. They are using the translation function, but I will create a separate PR for the keys.

**Comment by dannimajdalani** [comment]
Translation PR [#74](https://github.com/vanilla/locales/pull/74)

File: plugins/QnA/class.qna.plugin.php [hunk 1]
```diff
@@ -1378,4 +1378,16 @@ public function searchResultSchema_init(Schema $schema) {
         $types[] = 'question';
         $schema->setField('properties.type.enum', $types);
     }
+
+    /**
+     * Adds status notification options to profiles.
+     *
```
Comment (cchabilall83) [inline plugins/QnA/class.qna.plugin.php]: this doesn't look right

Comment (dannimajdalani) [reply plugins/QnA/class.qna.plugin.php]: This is a description of what follows.

**Pull request merged**